client/src/app/submission/submission.component.ts

```diff
@@ -58,6 +90,7 @@ export class SubmissionComponent implements OnInit, OnDestroy {
   archiveEntityDataSource: PaginatedDataSource<ArchiveEntity>;
 
   private MAX_ERRORS = 1;
+  SubmissionTab = SubmissionTab;
```

Comment:
What does this mean?

Reply:
This is need so that the SubmissionTab enum can be accessed in the template.

Reply:
@aaclan-ebi I think in this case this would be better:
`submissionTab = SubmissionTab;`

Reply:
ah sure, but how is that better than the current one?

Reply:
ah, i think it's because it became a variable now? actually, what's the convention for enums?

Reply:
Ah it's camelCase https://basarat.gitbook.io/typescript/styleguide#enum
But i kindof agree that it should be `submissionTab = SubmissionTab;` because I'm using it as variable.

Reply:
Done.